### PR TITLE
Fix: Minor changes and typo removal in "How Computers Speak"

### PR DESCRIPTION
**Pull request by marsbird**
This PR includes just a few small changes to [How Computers Speak](https://unicorn-utterances.com/posts/how-computers-speak), fixing various typos.

### CODE REVIEW

File: content/crutchcorn/posts/how-computers-speak/index.md [hunk 1]
```diff
@@ -231,7 +231,7 @@ int main() {
 }
 ```
 
-This code simply says, "print the number 185 to the screen so the user can see it". **To do the same in assembly requires a massive amount of knowledge about the system** you're intending to run code on, due to the lack of portability granted by higher-level languages.
+This code simply says, "print the number 185 to the screen so the user can see it". **To do the same in assembly requires a massive amount of knowledge about the system** you're intending to run code on, due to the lack of portability granted by lower-level languages.
```
Comment (crutchcorn) [inline content/crutchcorn/posts/how-computers-speak/index.md:234]: Let's revert this change or rephrase this sentence.

The intended message was "higher level languages has portability that assembly does not, leading to more knowledge required to write assembly"

**Review by crutchcorn — CHANGES_REQUESTED** [review]
Minor change requested. Otherwise, thanks for this!

File: content/crutchcorn/posts/how-computers-speak/index.md (hunk 1 above)
Comment (marsbird) [reply content/crutchcorn/posts/how-computers-speak/index.md:234]: Hi! Right now, the sentence says that higher-level languages have a "lack" of portability, implying that higher-level languages *aren't* as portable as lower-level languages, which is the opposite of the intended point. 
I'm happy to revert the change if you don't agree.

Comment (crutchcorn) [reply content/crutchcorn/posts/how-computers-speak/index.md:234]: For sure. I see both sides, but that ambiguity is probably a sign we should rewrite this sentence. 

Any chance you wanna tackle that real quick then we can merge? ☺️

Comment (marsbird) [reply content/crutchcorn/posts/how-computers-speak/index.md]: No problem!

File: content/crutchcorn/posts/how-computers-speak/index.md
Comment (marsbird) [inline content/crutchcorn/posts/how-computers-speak/index.md:1]: @crutchcorn thoughts on this rewrite?

**Review by crutchcorn — APPROVED** [review]
Love this! Thanks for your help!

Btw, are you in our Discord? 😄

**Comment by marsbird** [comment]
I'll join right now! I was just sent the Framework Field Guide by a friend and I'm loving all the resources you've got on here! Happy to help :)
